src/contributing/README.md

```diff
@@ -27,4 +27,7 @@ Signed-off-by: Full Name <email address>
 If you'd like to get a feel for what we're looking for in contributions, we also have a checklist
 for PR reviewers [here](pr_checklist.md).
 
+When planning to upstream a project, make sure to read our [package management and versioning
```

Comment:
Again we still have the "upstream" and "project" terms here, which I think we were moving away from.

Reply:
🤦🏻‍♂️ yes, need to change those as well